src/style.css

```diff
@@ -1,3 +1,87 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
 body {
+  padding: 0;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  text-align: center;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: inline-block;
+}
+
+.nav__link {
+  padding: 0;
+  line-height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  align-items: center;
+  display: inline-block;
+  position: relative;
```

Comment:
seems redundant here - you need it only in `is-active` and you actually add it in `.is-active`